pkg/operation/seed/seed.go

```diff
@@ -1728,7 +1753,7 @@ func switchIngressClass(ctx context.Context, seedClient client.Client, ingressKe
 	return seedClient.Update(ctx, ingress)
 }
 
-func computeNginxIngress(seed *Seed) map[string]interface{} {
+func computeNginxIngress(ctx context.Context, seed *Seed, seedClientSet kubernetes.Interface) (map[string]interface{}, error) {
```

Comment:
nginx-ingress was refactored with #5208 and is now deployed via `ManagedResource`. In such cases, existing annotations should be kept. Can you rebase to this PR and check if this is true?

Reply:
ah this explains now, why the helm chart was missing from the master I guess.
I will look into it, thx for the hint!

Reply:
Same here, you could adapt the resource-manager's merge func for services to keep annotations with the `loadbalancer.openstack.org/` prefix: https://github.com/gardener/gardener/blob/e14c573153af5ee57519d05c7935ba925da109b7/pkg/resourcemanager/controller/managedresource/merger.go#L284